src/server/utilities/canAdvertise/index.js

```diff
@@ -0,0 +1,15 @@
+const canAdvertise = (headers, override) => {
+  if (override) {
```

Comment:
Hey! I was more thinking we could check for a query string in the URL. Something like `force-ads=true` that we could check in here, which means that people can see ads for testing purposes without needing the correct headers to be set :)

Reply:
I've added a query param (with the suggested string above).

Ads can be seen locally if you visit this URL: http://localhost:7080/pidgin?force-ads=true&adUnit=/4817/bbccom.test.site.flash/news_homepage_int